src/components/dialog-full-screen/content.style.js

```diff
@@ -0,0 +1,26 @@
+import styled, { css } from 'styled-components';
+import FullScreenHeading from './full-screen-heading/full-screen-heading.style';
+import contentClassicStyle from './content-classic.style';
+
+const StyledContent = styled.div`
+  overflow-y: auto;
+
+  ${({ headingHeight }) => css`
+    ${FullScreenHeading} + & {
+      height: calc(100% - ${headingHeight}px);
+    }
+  `}
+
+  .carbon-app-wrapper {
```

Comment:
Is this class name necessary?

Reply:
Yes, app wrapper is not a styled component yet.